code/__DEFINES/traits.dm

```diff
@@ -104,6 +105,8 @@
 #define TRAIT_UNDEFIBBABLE 		"undefibbable"//human can't be revived
 #define TRAIT_HEALING_INFUSION	"healing_infusion"//greatly improves natural healing for xenos
 #define TRAIT_PSY_DRAINED		"psy_drained"//mob was drained of life force by a xenos
+#define TRAIT_STAGGERIMMUNE		"stagger_immunity" //Immunity to stagger
+#define TRAIT_SLOWDOWNIMMUNE	"slowdown_immunity" //Immunity to slowdown
```

Comment:
this trait is never added, it's for later?

Reply:
yeah